### DISCUSSION TITLE
Names of Sensors are too verbose

### DISCUSSION THREAD

**Discussion by GrimmiMeloni** [OStrama/weishaupt_modbus#77] [Q&A]
Since updating to 1.0.4 I see a cosmetic issue.
When adding Sensors to some widget, they are all prefixed with "WH Devicename". This makes names unnecessary lengthy, and creates visual clutter in dashboards. 

I am not sure if this is some kind of setting, but basically I would like to have the same experience when using single sensors like I get when adding an entire device (via the configuration).

Here's an example. This is what I get when I navigate to the integration, pick the Warmwasser device, and then click "add to dashboard" below the list of sensors.

<img width="415" alt="image" src="https://github.com/user-attachments/assets/2d4785e1-4fda-43ea-b38d-edf38d4af850">

And this is what I get when I replicate this by creating an Entities card on a given dashboard and add the 3 sensors to it.

<img width="491" alt="image" src="https://github.com/user-attachments/assets/e5793730-409c-46bf-a797-44191f188697">

**Comment by OStrama** [comment]
The names are generated in the following way:
- We provide a translation key, which is used to lookup the name of the entity according to the current language from translation files. We provide translations for English, German and Dutch, currently. When a different language is used, we fall back to English. The translated name is the one you see in the device panel
- Additionally, Home Assistant generates an Entity ID that consists of the platform ("sensor", "number", ..), the device name and the entity name.
- for some reason, HA seems to use a name directly derived from the entity ID as default for the entity card.
- We have to use the translation mechanism to provide different languages. This implies the automatic generated names, so the only possibility that I see is manual renaming of the entities in the entity cards.

**Reply by GrimmiMeloni** [reply]
OK. I guess we are out of luck then.
